loadtest/main.go

```diff
@@ -167,51 +171,65 @@ func run(config Config) {
 		for _, account := range activeAccounts {
 			key := GetKey(uint64(account))
 			var msg sdk.Msg
-			msgType := config.MsgTypeDistr.Sample()
-			orderPlacements := []*dextypes.Order{}
-			var orderType dextypes.OrderType
-			if msgType == "limit" {
-				orderType = dextypes.OrderType_LIMIT
-			} else {
-				orderType = dextypes.OrderType_MARKET
-			}
-			var direction dextypes.PositionDirection
-			if rand.Float64() < 0.5 {
-				direction = dextypes.PositionDirection_LONG
-			} else {
-				direction = dextypes.PositionDirection_SHORT
-			}
-			price := config.PriceDistr.Sample()
-			quantity := config.QuantityDistr.Sample()
-			contract := config.ContractDistr.Sample()
-			for j := 0; j < int(batchSize); j++ {
-				orderPlacements = append(orderPlacements, &dextypes.Order{
-					Account:           sdk.AccAddress(key.PubKey().Address()).String(),
-					ContractAddr:      contract,
-					PositionDirection: direction,
-					Price:             price.Quo(FromMili),
-					Quantity:          quantity.Quo(FromMili),
-					PriceDenom:        "SEI",
-					AssetDenom:        "ATOM",
-					OrderType:         orderType,
-					Data:              VortexData,
-				})
-			}
-			amount, err := sdk.ParseCoinsNormalized(fmt.Sprintf("%d%s", price.Mul(quantity).Ceil().RoundInt64(), "usei"))
-			if err != nil {
-				panic(err)
-			}
-			msg = &dextypes.MsgPlaceOrders{
-				Creator:      sdk.AccAddress(key.PubKey().Address()).String(),
-				Orders:       orderPlacements,
-				ContractAddr: contract,
-				Funds:        amount,
+			switch config.MessageType {
+			case "basic":
```

Comment:
nit: not a blocker but i think it might be nice to have these as helper methods so it's easier to read